superset-frontend/src/SqlLab/components/ResultSet/ResultSet.test.tsx

```diff
@@ -363,7 +368,7 @@ describe('ResultSet', () => {
       );
     });
     const { getByRole } = setup(mockedProps, mockStore(initialState));
-    expect(getByRole('treegrid')).toBeInTheDocument();
+    expect(getByRole('grid')).toBeInTheDocument();
```

Comment:
For some reason, in jest runtime, the ag grid root had a role "grid", not "treegrid"